### PR TITLE
Siva

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: Untitled [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# SEED system status
+
+This page provides the following Information:
```
Comment (yilunyou) [inline Untitled:3]: I will suggest to keep this page out of this pr until we have real data in it.

File: docs/_sidebar.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 - [Edit TechPass profile](edit-profile)
 - [Onboard to SEED](onboard-to-seed)
 - [Reset MFA](reset-mfa)
+- [System Status](status)
```
Comment (yilunyou) [inline docs/_sidebar.md:9]: same for this option. let's remove it from the options until there's real data.
I will suggest to put in the prod maintenance details if we really want to publish this page

File: docs/status.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# TechPass system status
+This page provides the TechPass system status, information on the following:
+- [Scheduled maintenance](#scheduled-maintenance)
+- [Ongoing incident(s)](#ongoing-incident-s)
+- [Previous incidents](#previous-incidents)
+
+## Scheduled maintenance
+Posted on 14 July 2022
```
[the remark this answers is not in the document]
Comment (yilunyou) [reply docs/status.md:8]: @yqyeoh please provide siva with the details when you have them

File: docs/whats-new/additional-resources.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Additional resources for service changes
```
Comment (yilunyou) [inline docs/whats-new/additional-resources.md:1]: hmmm..I don't think we should have this section is separated out as additional resources here in what's new..

2 things:
1. the content of this is a guide for only tenant admins. so it should only be in tenant guide
2. this guide is not a temporarily content. it's something useful like a blog. like the section in concepts: https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/concepts/oauth2. I will prefer such guides be treated as guides and not as additional resources that can only be found in what's new. content in what's news should point to the guides residing in more permanent locations

**Review by yilunyou — CHANGES_REQUESTED** [review]

**Review by yilunyou — CHANGES_REQUESTED** [review]

File: docs/status.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (yqyeoh) [reply docs/status.md:8]: @siva the details for prod release 27 Jul 2022 would be the same as what we have for staging release 20 July 2022 (including the FE & BE versions), except for the following:
- replace `https://api.stg.techpass.suite.gov.sg` with `https://api.techpass.gov.sg`
- replace `https://api.stg.techpass.suite.gov.sg/.default` with `https://api.techpass.gov.sg/.default`

File: Untitled (hunk 1 above)
Comment (SenthilKumarSivakami) [reply Untitled:3]: renamed the file.

File: docs/_sidebar.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/_sidebar.md:9]: noted

File: docs/status.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply docs/status.md:8]: Noted. Thanks.

File: docs/techpass-status.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# TechPass system status
+This page provides the status of the TechPass service and information on the following:
+- [Scheduled maintenance](#scheduled-maintenance)
+- [Ongoing incidents](#ongoing-incidents)
+<!--- [Previous incidents](#previous-incidents)-->
+
+## Scheduled maintenance
+Posted on 26 July 2022
+
+**Scheduled** - TechPass portal will be going through a migration on 26 July 2022. Public officers may not be able to access TechPass portal during this time. There is no action required from you.
```
Comment (yilunyou) [inline docs/techpass-status.md:10]:
```suggestion
**Scheduled** - TechPass portal will be going through a scheduled maintenance on 26 July 2022, 1030am. There may be connectivity issues during this time. There are no action required from you.
```

File: docs/whats-new/staging-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+<kbd>![developer portal widget](../assets/images/whats-new/20220706_masthead-devportalwidget-02.png)</kbd>
+
+</details>
+
+<details>
+  <summary style="font-size:20px;font-weight:bold">New webhook in TechPass portal</summary>
+
+Tenants can configure a new event webhook, `application-deleted` to get notifications when an application gets deleted from their system.
+
+For more information, refer to [Configuring Webhooks](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/webhooks?id=configuring-your-webhooks-).
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:56]:
```suggestion
For more information, refer to [Configuring Webhooks](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/webhooks?id=configuring-your-webhooks).
```

File: docs/whats-new/staging-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+</details>
+
+<details>
+  <summary style="font-size:20px;font-weight:bold">New webhook in TechPass portal</summary>
+
+Tenants can configure a new event webhook, `application-deleted` to get notifications when an application gets deleted from their system.
+
+For more information, refer to [Configuring Webhooks](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/webhooks?id=configuring-your-webhooks-).
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:65]:
```suggestion
For more information, refer to [Configuring Webhooks](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/webhooks?id=configuring-your-webhooks).
```

File: docs/whats-new/staging-release-notes.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+
+  **Earlier**:
+
+  Tenants were unable to find users by mobile number.They were not able to determine if the same user has attempted to request for a new account or if it was a suspicious attempt.
+
+  **Now**:
+  Tenants can search for users using their mobile number. This feature comes in handy for tenants to identify if a user already has an account in the TechPass system and if yes, invitation will not be sent to that user.
+
+  For more information, refer to the following documentation:
+  - [List Users By Namespace](https://docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1namespace~1{namespace}~1users/get)
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:133]:
```suggestion
  - [List Users By Namespace](https://stg.docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1namespace~1{namespace}~1users/get)
```

File: docs/whats-new/staging-release-notes.md [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+  - [List Users](https://docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1users/get).
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:134]:
```suggestion
  - [List Users](https://stg.docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1users/get).
```

File: docs/whats-new/staging-release-notes.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+  </summary>
+  </details>
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Three different login timestamps available</summary>
+
+[Get User Info API](https://docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1users~1{identifier}/get) now lists the following three different last login date and time:
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:141]:
```suggestion
[Get User Info API](https://stg.docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1users~1{identifier}/get) now lists the following three different last login date and time:
```

**Review by yilunyou — CHANGES_REQUESTED** [review]

File: docs/whats-new/production-release-notes.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+<kbd>![developer portal widget](../assets/images/whats-new/20220706_masthead-devportalwidget-02.png)</kbd>
+
+</details>
+
+<details>
+  <summary style="font-size:20px;font-weight:bold">New webhook in TechPass portal</summary>
+
+Tenants can configure a new event webhook, `application-deleted` to get notifications when an application gets deleted from their system.
+
+For more information, refer to [Configuring Webhooks](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/webhooks?id=configuring-your-webhooks-).
```
Comment (yilunyou) [inline docs/whats-new/production-release-notes.md:39]:
```suggestion
For more information, refer to [Configuring Webhooks](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/webhooks?id=configuring-your-webhooks).
```

File: docs/whats-new/staging-release-notes.md [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Parameter changed in the request for access token</summary>
+
+There is a change to the `scope` parameter in the request for access token via client credentials grant.
+
+**Action required**: Change the `scope` parameter value from `https://graph.microsoft.com/.default` to `https://api.techpass.gov.sg/.default`.
+
+For more information, refer to the following:
+- [Transition guide](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/concepts/transition-guide)
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:15]:
```suggestion
- [Transition guide](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/#/concepts/transition-guide)
```

**Review by yilunyou — APPROVED** [review]

**Pull request merged**